lib/notifiers/notifiers.go

```diff
@@ -322,13 +364,51 @@ func getGCSConfig(ctx context.Context, grf gcsReaderFactory, path string) (*Conf
 	return cfg, nil
 }
 
+// getGCSConfig fetches the Template file from the given GCS path and returns the parsed Config.
+func getGCSTemplate(ctx context.Context, grf gcsReaderFactory, path string) (string, error) {
+	if trm := strings.TrimPrefix(path, "gs://"); trm != path {
+		// path started with the prefix
```

Comment:
it seems like you can just use a regex for this.

Reply:
done